src/server/game/Chat/Channels/Channel.cpp

```diff
@@ -294,6 +297,8 @@ void Channel::LeaveChannel(Player* player, bool send)
         SendToAll(builder);
     }
 
+    sIRC->Handle_WoW_Channel(_name, ObjectAccessor::FindPlayer(guid), CHANNEL_LEAVE);
```

Comment:
deserves to be implemented as script hook (if not existing, add one on join/leave channel)